metrics/seqeval/README.md

```diff
@@ -0,0 +1,139 @@
+# Metric Card for seqeval
+
+## Metric description
+
+seqeval is a Python framework for sequence labeling evaluation. seqeval can evaluate the performance of chunking tasks such as named-entity recognition, part-of-speech tagging, semantic role labeling and so on. 
+
+
+## How to use 
+
+Seqeval produces labelling scores along with its sufficient statistics from a source against one or more references.
+
+It takes two mandatory arguments:
+
+`predictions`: a list of lists of predicted labels, i.e. estimated targets as returned by a tagger.
+
+`references`: a list of lists of reference labels, i.e. the ground truth/target values.
+
+It can also take several optional arguments:
+
+`suffix` (boolean): `True` if the IOB prefix is after type, `False` otherwise. The default value is `False`.
```

Comment:
Maybe worth rephrasing it?
```suggestion
`suffix` (boolean): `True` if the IOB tag is a suffix (after type) instead of a prefix (before type), `False` otherwise. The default value is `False`, i.e. IOB tag is a prefix (before type).
```